packages/react/src/components/TooltipTabstop/index.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TooltipTabstop from './';
+import axe from '../../axe';
+
+test('should render without errors', () => {
+  render(<TooltipTabstop tooltip="World">Hello</TooltipTabstop>);
+  expect(screen.getByText('Hello')).toBeInTheDocument();
+});
+
+test('should display tooltip on hover', async () => {
+  render(<TooltipTabstop tooltip="World">Hello</TooltipTabstop>);
+
+  await userEvent.hover(screen.getByText('Hello'));
+  expect(screen.getByText('World')).toBeInTheDocument();
```

Comment:
If you look at how we structure the [tooltip tests](https://github.com/dequelabs/cauldron/blob/0efb20782bc2943a438d9810d065cdf0af93efd1/packages/react/src/components/Tooltip/Tooltip.test.tsx#L67-L73) we likely want these to be very similar to that: 

```suggestion
  expect(screen.queryByRole('tooltip')).not.toBeInTheDocument();
  await fireEvent.focusIn(screen.getByRole('button'));
  expect(screen.queryByRole('tooltip')).toBeInTheDocument();
```

Reply:
Additionally, we want to validate the accessible description as well:

```tsx
expect(
  screen.getByRole('button', { name: 'button' })
).toHaveAccessibleDescription('World');
```